_inc/client/at-a-glance/backups.jsx

```diff
@@ -232,7 +232,7 @@ class DashBackups extends Component {
 		return (
 			<div>
 				<QueryVaultPressData />
-				{ 'unavailable' === this.props.rewindStatus ? (
+				{ 'unavailable' === this.props.rewindStatus || '' === this.props.rewindStatus ? (
```

Comment:
This should be fixed regardless. It's what causes the non-existent Rewind/backups card on some errors in the AAG dashboard.

Reply:
https://github.com/Automattic/jetpack/pull/16867